src/dotVariant.Generator/RenderInfo.cs

```diff
@@ -201,13 +201,15 @@ public sealed record ParamInfo(
             var emitNullable = desc.NullableContext.HasFlag(NullableContext.AnnotationsEnabled);
             var disposable = compilation.GetTypeByMetadataName(typeof(IDisposable).FullName)!;
 
+            const string attributeName = nameof(dotVariant) + "." + nameof(NoImplicitConversionAttribute);
+
             var paramDescriptors =
                 desc
                 .Options
                 .Select((p, i) => new ParamInfo(
                     CanBeNull: CanBeNull(p, desc.NullableContext),
                     DiagType: p.Type.WithNullableAnnotation(p.NullableAnnotation).ToDisplayString(DiagFormat),
-                    EmitImplicitCast: !(p.Type.TypeKind == TypeKind.Interface || IsAncestorOf(p.Type, desc.Type)),
+                    EmitImplicitCast: !(p.Type.TypeKind == TypeKind.Interface || IsAncestorOf(p.Type, desc.Type) || p.GetAttributes().Any(a => a.AttributeClass?.ToDisplayString() == attributeName)),
```

Comment:
Please move `p.GetAttributes().Any(a => a.AttributeClass?.ToDisplayString() == attributeName)` to a separate function in `Inspect.cs` called `HasImplicitConversionDisabled(IParameterSymbol p)`.

Reply:
Wasn't sure if I should do that, thought it was too specific, comparing to the other tests^^